corona-warn-app/cwa-app-android
# Notification for g status change (EXPOSUREAPP-12061)

[Ticket](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12061)

When testing, make sure to force trigger the CCL config update from the test menu after changing testing environments. Otherwise the CCL config will remain cached with the config from the previous environment.

How to test:


Switch to 'MOCK-CLOUD' environmnet
Scan an old 1/2 certificate: 
`cwa vc gen --pii 42 -s v.0.dn=1 v.0.sd=2 v.0.mp='EU/1/20/1528' v.0.dt='2021-02-01'`
Scan an old 2/2 certificate:
` cwa vc gen --pii 42 -s v.0.dn=2 v.0.sd=2 v.0.mp='EU/1/20/1528' v.0.dt='2021-03-01'`
Scan a new 2/2 certificate
`cwa vc gen --pii 42 -s v.0.dn=2 v.0.sd=2 v.0.mp='EU/1/20/1528' v.0.dt='2022-02-01'`
 
You should see the red badge next to the "Proof of status" card whenever the admission state changes.
Pressing the back button or the "X" button on the `PersonDetailsScreen` will dismiss the notification.

TODO:
